test/verify/check-metrics

```diff
@@ -1261,6 +1265,19 @@ BEGIN {{
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='In']", "0")
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='Out']", "0")
 
+        b.click("[aria-label='Network usage'] [data-interface='lo'] button")
+        b.enter_page("/network")
+        b.wait_visible("#network-interface-name:contains('lo')")
+        if "ubuntu" in m.image:
+            b.wait_in_text("#network-interface-settings", "This device cannot be managed here.")
```

Comment:
This is one of the cases where it differs at the moment

Reply:
Yes, please add an `else` branch here to at least pin down the current behavior.

Reply:
Done